modules/delay-hydration.yml

```diff
@@ -1,7 +1,7 @@
 name: delay-hydration
 description: Improve your Nuxt.js v2 Google Lighthouse score by delaying hydration.
 repo: harlan-zw/nuxt-delay-hydration
-npm: nuxt-delay-hydration
+npm: mono-nuxt-delay-hydration
```

Comment:
```suggestion
npm: nuxt-nuxt-delay-hydration
```

Reply:
this has to be fixed in main. `repo` field needs update